src/cms/templates/dashboard/admin_dashboard.html

```diff
@@ -1,5 +1,32 @@
 {% extends "_base.html" %}
 {% load i18n %}
 {% block content %}
-    Admin Dashboard
-{% endblock %}
\ No newline at end of file
+{% load static %}
+{% load widget_tweaks %}
+   
+   <h2>{% trans 'Technical Feedback' %}</h2>
+
+<table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+    <tr class="border-b border-solid border-gray-200">
+        <th class="text-sm text-left uppercase py-3">{% trans 'Language' %}</th>
+        <th class="text-sm text-left uppercase py-3">{% trans 'Comment' %}</th>
+        <th class="text-sm text-left uppercase py-3">{% trans 'Rating' %}</th>
+        <th class="text-sm text-right uppercase py-3 pr-2">{% trans 'Date' %}</th>
+    </tr>
+  {% if not  admin_feedback_exists %}
+    <tr><td colspan="6" class="px-2 py-3">{% trans 'No Feedback available yet.' %} </td></tr>
+    
+  {% else%} 
+    {% for feedback in all_feedback %}
+     <tr>
+        
+       <td class="pl-2 py-3">{{ feedback.language.translated_name }}</td>
+        <td>{{ feedback.comment }}</td>
```

Comment:
Maybe we can omit the comment field here and just display the emotion? @ulliholtgrave do you have an opintion, as you're "in charge" of the dashboard.

Reply:
Ignore that, I just realized that this PR currently is about the admin dashboard.

Reply:
So far, I only have a styled grid with the different spaces for the content. Maybe we can make this field on_click()-expandable with a shortened version of the comment, what do you think?

Reply:
Also possible, yes. I think we should regard this as non-blocking and iterate later.